.travis.yml

```diff
@@ -1,35 +1,51 @@
+language: python
+
 notifications:
   email: false
 
-language: python
-python:
-    - "2.7"
+matrix:
+  include:
+    - os: linux
+      python: 2.7
+      env: MINICONDA_NAME=Miniconda2-latest-Linux-x86_64.sh
+      sudo: required
+    - os: osx
+      language: generic
+      env: MINICONDA_NAME=Miniconda2-latest-MacOSX-x86_64.sh
 
-sudo: false
-
-before_install:
-    # Setup anaconda
-    - wget http://repo.continuum.io/miniconda/Miniconda-latest-Linux-x86_64.sh -O miniconda.sh
-    - chmod +x miniconda.sh
+install:
+    - wget http://repo.continuum.io/miniconda/${MINICONDA_NAME} -O miniconda.sh;
     - bash miniconda.sh -b -p $HOME/miniconda
     - export PATH="$HOME/miniconda/bin:$PATH"
     - conda update --yes conda
-
-install:
-    - conda env create --file environment.yml
+    - conda env create --file environment_${TRAVIS_OS_NAME}.yml
     - source activate py2_parcels
     - conda install --yes sphinx
     - pip install -e .
 
-before_script: # configure a headless display to test  plot generation
-    - "export DISPLAY=:99.0"
-    - "sh -e /etc/init.d/xvfb start"
-    - sleep 3 # give xvfb some time to start
-
 script:
-    - flake8 parcels
-    - flake8 tests
-    - parcels_get_examples examples/
+    - |
+      # Set up display to be able to plot in linux
+      if [[ "${TRAVIS_OS_NAME}" = "linux" ]]; then
+        export DISPLAY=:99.0;
+        sh -e /etc/init.d/xvfb start;
+        sleep 3;
+      fi
     - py.test -v -s tests/
-    - py.test -v -s --nbval-lax examples/
-    - make linkcheck -C docs
+    - parcels_get_examples examples/
```

Comment:
parcels_get_examples is only required when examples is actually run; i.e. only on linux. Looking at travis output, even mac still downloads the data but doesn't seem to do anything with it

Reply:
I originally left this switched on intentionally: I wanted to make sure that at least downloading the examples also works for OSX. But that's essentially amounting to checking whether the entry point in the setup.py is working.

I think we could make this linux-only as well.